docs/source/common/optimizers.rst

```diff
@@ -21,46 +21,101 @@ Manual optimization
 For advanced research topics like reinforcement learning, sparse coding, or GAN research, it may be desirable
 to manually manage the optimization process. To do so, do the following:
 
-* Disable automatic optimization in Trainer:  Trainer(automatic_optimization=False)
+* Disable automatic optimization in Trainer:  Trainer(automatic_optimization=False) or override your LightningModule ``automatic_optimization`` property to False
```

Comment:
The Trainer property is deprecated I think, we shouldn't advertise it then

Reply:
great catch !